### DISCUSSION TITLE
Disable rke2-ingress-nginx doesn't work properly

### DISCUSSION THREAD

**Discussion by Exobitt** [rancher/rke2#4625] [Q&A] [answered]
**Environmental Info:**
RKE2 Version: 
v1.27.4+rke2r1

Cluster Configuration:
3 servers, 3 workers

**Describe the bug:**
Appending the:
```yaml
disable:
    - rke2-ingress-nginx
```
Doesn't disable rke2-ingress-nginx. But after a restart of rke2, rke2 terminates the pods

**Steps To Reproduce:**
- Installed RKE2: 
- Include the disable: rke2-ingress-nginx in your config
- After everything is setup, restart the service

**Expected behavior:**
It shouldn't start the pods of the nginx ingress controller

**Actual behavior:**
It starts the pods of the ingress controller

**Comment by brandond** [comment, marked as answer]
It sounds like you're modifying the configuration after RKE2 has already been started, and rke2-ingress-nginx isn't disabled until after you restart the services and the new configuration is applied.

Ensure that you're configuring RKE2 before starting it the first time, and that you have added identical `disable` configuration on all three servers.

**Reply by Exobitt** [reply]
Its added before the start of rke2.
Though, im unsure whether or not im only applying it to the first server node.
I'm checking up on this in the morning.
Thank you for your answer!
